### ISSUE TITLE
Support for global stats and achievements

### ISSUE THREAD

**Issue by 1-800-thewolf** [YoYoGames/GMEXT-Steamworks#54]
### Is your feature request related to a problem?

Yes, I want to use the Steam API to handle global stats and achievements. I'm unable to use the Steam-provided tools without Gamemaker support here.
An example of a function I want to use is the following:
GML version: steam_get_achievement_achieved_percent
Steam version: GetAchievementAchievedPercent

### Describe the solution you'd like

I want GML to handle steam global functions. Here's steam's AUP page which covers Global stats and achievements:
https://partner.steamgames.com/doc/api/ISteamUserStats

Here's Gamemaker's Steamworks wiki page on Stats and Achievements which doesn't describe global stats or achievements at all:
https://github.com/YoYoGames/GMEXT-Steamworks/wiki/achievements

If GML can already use these functions, we need documentation on how to use it properly.

### Describe alternatives you've considered

I've searched the GML Steamworks wiki, the internet, and forms for how to use these functions and haven't found any.

I've also tried using steam_get_achievement_achieved_percent and inputting an achievement in but it always returns 0.

### Additional context

My latest variation using trial and error to try to get this working is the following:

```
glob_stat=steam_request_global_stats(30);
show_debug_message("glob_stat: "+string(glob_stat));
glob_ach=steam_request_global_achievement_percentages();
show_debug_message("glob_ach: "+string(glob_ach));

have_ach=steam_get_achievement("ach_plot_of_land")

perc_ach=steam_get_achievement_achieved_percent("ach_plot_of_land")

show_debug_message("have_ach: "+string(have_ach))
show_debug_message("perc_ach: "+string(perc_ach))
```

**Unfortunately, this results in the following output:**
_Received stats and achievements from Steam
Audio group 1 -> Loaded
glob_stat: 1
glob_ach: 1
GetAchievementAchievedPercent Called
have_ach: 1
perc_ach: 0
steam_request_global_achievement_percentages? 2 
steam_request_global_stats SUCCESS: False_

If it were running correctly, _perc_ach_ should be greater than 0 because I have the achievement (and I'm the only player in the world so far as the game is unrelease as of now)

It also looks like it fails to request the global stats.

**Comment by tabularelf** [comment]
> This feature is already supported. Here's the solution according to official documentation:

Given that I see "ChatGPT" there, I already can tell you right now that this isn't the right documents. What op is asking for, the Steamworks extension only covers highscores. Not stats and achievements (like the ones that you can see globally on Steam normally)

**Comment by DiasFranciscoA** [comment]
There was a bug with:
`steam_get_achievement_achieved_percent` that is now fixed in 1.6.0 release on github

Please make requests into separate tickets.
